templates/components/form/item_itilobject_group.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<table class='tab_cadre_fixe'>
+    <tr class='tab_bg_1'>
+        <th> {{ __('Last tickets') }} </th>
+    </tr>
+    <tr class='tab_bg_1'>
+        <td class='text-start'>
+            {{ __('Child groups') }}
+            {% do call('Dropdown::showYesNo', [
```

Comment:
Best to use the dropdownYesNo macro for this. Currently, the "Child groups" text isn't a proper label for the field which would cause accessibility issues.